docs/dev/porting_legacy_tests.md

```diff
@@ -0,0 +1,90 @@
+# A Guide to Porting Original Ethereum Tests to EEST
+
+## Background
+
+EEST is the successor to [ethereum/tests](https://github.com/ethereum/tests) (aka "original tests"), a repository that defined EVM test cases from the [Frontier](https://ethereum.org/en/history/#frontier) phase up to and including [The Merge](https://ethereum.org/en/history/#paris). These test cases are specified as YAML (and occasionally JSON) files in the [`./src/`](https://github.com/ethereum/tests/tree/develop/src) sub-directory. JSON test fixtures, which are fully-populated tests that can be executed against clients, are generated using [ethereum/retesteth](https://github.com/ethereum/retesteth). These JSON artifacts are regenerated when needed and added to the repository, typically in the [`./GeneralStateTests/`](https://github.com/ethereum/tests/tree/develop/GeneralStateTests) sub-directory.
+
+From [Shanghai](https://ethereum.org/en/history/#shapella) onward, new test cases — especially for new features introduced in hard forks—are defined in Python within EEST. While the existing test cases remain important for client testing, porting ethereum/tests to EEST will help maintain and generate tests for newer forks. This also ensures feature parity, as client teams will only need to obtain test fixture releases from one source.
+
+While automating the conversion of the remaining YAML (or JSON) test cases to Python is possible, manually porting individual test cases offers several benefits:
+
+- Reducing the number of test cases by combining multiple YAML (or JSON) cases into a single Python test function using parametrization.
+- Potentially improving coverage by parametrizing the Python version.
+- Producing higher quality code and documentation, which are typically clearer than an automated conversion.
+- Ensuring better organization of tests within the `./tests` folder of execution-spec-tests by fork and EIP.
+
+## Porting an original test
+
+1. Select one or more YAML test cases from ethereum/tests to port and create an issue in this repository AND comment on [this tracker issue.](https://github.com/ethereum/execution-spec-tests/issues/972)
+
+2. [Add a new test](../writing_tests/index.md) in the appropriate fork folder, following the guidelines for [choosing a test type.](../writing_tests/types_of_tests.md#deciding-on-a-test-type)
+
+3. Submit a PR with the ported tests:
+
+   1. Add the list of ported YAML files to [`converted-ethereum-tests.txt`](https://github.com/ethereum/execution-spec-tests/blob/1b30c336eae6b0746ea4db441ac74406f2fb2322/converted-ethereum-tests.txt).
+   2. Open a PR to remove the ported tests from the _original tests_ repository.
```

Comment:
Additionally, I think we can write and link to `ethereum/tests` here instead of referring to "original tests".